aws/resource_aws_instance.go

```diff
@@ -1095,7 +1095,7 @@ func resourceAwsInstanceUpdate(d *schema.ResourceData, meta interface{}) error {
 				},
 			})
 			if err != nil {
-				if isAWSErr(err, "InvalidParameterCombination", "") {
+				if !isAWSErr(err, "InvalidParameterCombination", "") {
```

Comment:
Nit: do you mind adding back the comment? 
```
// Tolerate InvalidParameterCombination error in Classic, otherwise
// return the error
```

Reply:
thanks @DrFaust92! just one more run of `go fmt` should do the trick to handle the lint check resuslts :)